models/SkillModel.js

```diff
@@ -1,7 +1,24 @@
-var mongoose = require("mongoose");
+const mongoose = require("mongoose");
+const { constants } = require("../helpers/constants");
 
-var SkillSchema = new mongoose.Schema({
-    title: { type: String, required: true }
-})
+const Schema = mongoose.Schema;
+const DataTypes = Schema.Types;
 
-module.exports = mongoose.model("Skill", SkillSchema);
+const SkillsSchema = new Schema(
+  {
+    title: {
+      type: DataTypes.String,
+      required: true,
+    },
+    expertiseLevel: {
+      type: DataTypes.String,
+      enum: Object.values(constants.expertiseLevel),
+      default: constants.expertiseLevel.FRESHER,
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 3 tabs but found 6 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
			default: constants.expertiseLevel.FRESHER,
```